internal/cli/atlas/deployments/options/deployment_opts.go

```diff
@@ -52,6 +83,15 @@ type DeploymentOpts struct {
 	s              *spinner.Spinner
 }
 
+type Deployment struct {
+	Type            string
+	Name            string
+	MongoDBVersion  string
+	StateName       string
+	MongoDContainer *podman.Container
+	MongoTContainer *podman.Container
```

Comment:
do we need these 2 fields? 
I see that only the ID is used for starting/unpausing/stopping containers but this can be done using the container names too, which will be consistent with the rest of the podman calls.
For container names we have opts.LocalMongodHostname() and opts.LocalMongotHostname()

Reply:
updated